docs/platforms/apple/common/index.mdx

```diff
@@ -48,7 +48,7 @@ brew install getsentry/tools/sentry-wizard && sentry-wizard -i ios
 ```
 
 This will patch your project and configure the SDK. You'll only need to patch the project once, then you'll be able to add the patched files to your version control system.
-If you prefer, you can also [set up the SDK manually](/platforms/react-native/manual-setup/manual-setup/) or follow the instructions below to adapt the [configuration](#configure).
+If you prefer, you can also [set up the SDK manually](/platforms/apple/guides/ios/manual-setup/) or follow the instructions below to adapt the [configuration](#configure).
```

Comment:
```suggestion
If you prefer, you can also <PlatformLink to="/manual-setup/">set up the SDK manually</PlatformLink> or follow the instructions below to adapt the [configuration](#configure).
```

Reply:
`l`: I changed it to use the platform aware <PlatformLink> component - right now makes no difference since this is only shown for the iOS guide